generative_ai/embedding_batch_test.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import backoff
+
+from google.api_core.exceptions import ResourceExhausted
+
+import pytest
+
+import embedding_batch
+
+
+@backoff.on_exception(backoff.expo, ResourceExhausted, max_time=10)
+@pytest.fixture(scope="session", autouse=True)
+def test_embed_text_batch() -> None:
+    os.environ["GCS_OUTPUT_URI"] = "gs://python-docs-samples-tests/"
```

Comment:
As mentioned earlier, please define this in code samples itself.

Reply:
Since this is a write action, users need to provide their own bucket.